### ISSUE TITLE
pipx install mitmproxy fails with UnicodeEncodeError: 'charmap' codec can't encode character '\u2502' in position 112:

### ISSUE THREAD

**Issue by charlbury** [pypa/pipx#819]
**Describe the bug**
When running `pipx install mitmproxy` in a windows 10 command terminal, the install fails with the following errors

```
C:\Users\admin>pipx install mitmproxy

Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\Scripts\pipx.exe\__main__.py", line 7, in <module>
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 779, in cli
    return run_pipx_command(parsed_pipx_args)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 202, in run_pipx_command
    return commands.install(
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\commands\install.py", line 60, in install
    venv.install_package(
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\venv.py", line 238, in install_package
    subprocess_post_check_handle_pip_error(pip_process)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\util.py", line 349, in subprocess_post_check_handle_pip_error
    print(completed_process.stderr, file=pip_error_fh, end="")
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2502' in position 112: character maps to <undefined>
```

**How to reproduce**

Enter the command `pipx install mitmproxy` in a windows 10 command terminal

Results of above command with --verbose option
```
C:\Users\admin>pipx install mitmproxy --verbose
pipx >(setup:729): pipx version is 1.0.0
pipx >(setup:730): Default python interpreter is 'C:\\Users\\admin\\AppData\\Local\\Programs\\Python\\Python310\\python.exe'
pipx >(package_name_from_spec:323): Determined package name: mitmproxy
pipx >(package_name_from_spec:324): Package name determined in 0.0s
creating virtual environment...
pipx >(run_subprocess:172): running C:\Users\admin\AppData\Local\Programs\Python\Python310\python.exe -m venv --without-pip C:\Users\admin\.local\pipx\venvs\mitmproxy
pipx >(run_subprocess:172): running C:\Users\admin\.local\pipx\venvs\mitmproxy\Scripts\python.exe -c import sysconfig; print(sysconfig.get_path('purelib'))
pipx >(run_subprocess:172): running C:\Users\admin\.local\pipx\shared\Scripts\python.exe -c import sysconfig; print(sysconfig.get_path('purelib'))
pipx >(run_subprocess:172): running C:\Users\admin\.local\pipx\venvs\mitmproxy\Scripts\python.exe --version
pipx >(_parsed_package_to_package_or_url:128): cleaned package spec: mitmproxy
installing mitmproxy...
pipx >(run_subprocess:172): running C:\Users\admin\.local\pipx\venvs\mitmproxy\Scripts\python.exe -m pip install mitmproxy
pipx >(subprocess_post_check_handle_pip_error:334): 'C:\\Users\\admin\\.local\\pipx\\venvs\\mitmproxy\\Scripts\\python.exe -m pip install mitmproxy' failed

pipx >(rmdir:55): removing directory C:\Users\admin\.local\pipx\venvs\mitmproxy
Traceback (most recent call last):
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\Scripts\pipx.exe\__main__.py", line 7, in <module>
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 779, in cli
    return run_pipx_command(parsed_pipx_args)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\main.py", line 202, in run_pipx_command
    return commands.install(
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\commands\install.py", line 60, in install
    venv.install_package(
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\venv.py", line 238, in install_package
    subprocess_post_check_handle_pip_error(pip_process)
  File "C:\Users\admin\AppData\Roaming\Python\Python310\site-packages\pipx\util.py", line 349, in subprocess_post_check_handle_pip_error
    print(completed_process.stderr, file=pip_error_fh, end="")
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2502' in position 112: character maps to <undefined>
```

**Expected behavior**

mitmproxy is successfully installed

**Comment by cs01** [comment]
Thanks for the report.

I believe this was fixed in https://github.com/pypa/pipx/pull/811, and should work on the `main` branch if you want to try it out. 

It will be included in the next release of pipx.

**Comment by joaoaccaldeira** [comment]
@cs01 
Bit of a noobie here, Im having the same problem with 'pipx install eth-brownie'.

How would I go about installing the code in the main branch to my computer?

Really would like to be able to continue doing the stuff I was doing and not sure how long it will take you to issue the next release 😢 

Thanks in advance!

**Comment by charlbury** [comment]
Thanks for the update. Apologies but how do I do an install of pipx using that specific branch?

**Comment by meowmeowmeowcat** [comment]
```
pipx install git+https://github.com/pypa/pipx.git@main
```

should work.

**Comment by cs01** [comment]
This has been fixed in v1.1.0
